index.html

```diff
@@ -77,6 +77,230 @@ <h3>LET'S CONNECT</h3>
         </li>      
        </ul>
     </section>
+    <section class="portfolio" id="portfolio">
+      <nav class="portfolio-border">
+        <img
+          class="portfolio-image"
+          alt="portfolio project"
+          src="Snapshoot_Portfolio1.png"
+        />
```

Comment:
- [ ] I must commend your design as it looks very good, you have done very following the Figma design. However, I notice a few issues with the page layout. The screenshot below is your project displayed on a screen size of `375px`. As you can see below, there are unwanted margins on both left and right sides and this is because of a couple of things I have tried to identify below:

- #### You designed the mobile version on a desktop screen size instead of using the browser mobile dimension. 
 > To activate the mobile screen, right-click anywhere on your page, click on inspect and set the dimension at the top to `375px`.
 
- #### You declared the following on the `body` element.
     - `display: flex;` 
     - `justify-content: center;` 
- #### You declared the following on most of the child element of `<section class="portfolio" id="portfolio">`.
     - `width: 327px;` Please [click here](https://www.thoughtco.com/fixed-width-vs-liquid-layouts-3468947) to  read about why you should not use a fixed width for your layout: 

👉 Essentially, your design should cover the whole space view-port when using a screen size of  `375px`. Kindly re-work your CSS to fix this issue. if you must declare width, use `100%` or percentage value.

Your design |
--- |
![image](https://user-images.githubusercontent.com/84156714/185743426-a5e82345-fcac-418b-a763-6348770842aa.png)

Reply:
Good job following the previous reviewer's suggestions. but you could improve a little more, notice that it has a rounded corner in the lower left, you also need to repair the GitHub icon link